lib/src/logger/loggers/void_logger.dart

```diff
@@ -7,7 +7,7 @@ class VoidLogger extends Logger {
   VoidLogger() : super(LogLevel.debug);
 
   @override
-  int? get wrapTextColumn => null;
+  int? get wrapTextColumn => 80;
```

Comment:
Question: Why is this required for the void logger?

I don't think there is a guarantee that this will be set in a real environment so the code might need to handle this case.

Reply:
Ah good catch, this was a left over from debugging 🙏